<launch>

  <arg name="sim" default="false"/>
  <arg name="timeout" default="90.0"/>

  <node name="pregrasp_server" pkg="amazon_challenge_motion" type="pregrasp_server" output="screen" respawn="true">
    <param name="timeout" value="$(arg timeout)"/>
    <param name="sim" value="$(arg sim)"/>
  </node>

  <group if="$(arg sim)">
    <node name="action_client" pkg="amazon_challenge_bt_actions" type="ActionClient" output="screen">
      <param name="action_server_name" value="pregrasp_server" />
    </node>
    <node name="task_manager" pkg="amazon_challenge_bt_actions" type="task_manager" output="screen"/>
  </group>

</launch>